app/src/main/java/com/example/project_movie_01/data/model/GenresMovie.java

```diff
@@ -3,7 +3,7 @@
 import com.example.project_movie_01.constant.Constant;
 import com.example.project_movie_01.utils.StringUtils;
 
-public class PopularMovie {
+public class GenresMovie {
```

Comment:
```suggestion
public class MovieGenre {
```
hoặc chỉ Genre thôi. Update những chỗ khác chứa GenresMovie